### DISCUSSION TITLE
Find a hidden word

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#621] [Galaxy Trivia] [answered]
<img width="263" alt="image" src="https://user-images.githubusercontent.com/5120290/234347876-8d4093e1-0c12-4ff7-93d8-57047699a6ed.png">

**Comment by MohammedNayeem9** [comment]
The hidden word is 
**CONSOLE**

**Comment by nsafarova** [comment, marked as answer]
Console!

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/nsafarova?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by gotthejuicee** [comment]
ITS Console!

**Reply by fasakinhenry** [reply]
I can't seem to locate it
